js/vue/meetupEvents.vue

```diff
@@ -0,0 +1,96 @@
+
+<template>
+  <div>
+    <div class="meetups">
+      <div class="meetup-event" v-for="event in events" :key="event.id">
+        <h3 class="meetup-heading"> {{ event.name }}</h3>
+        <hr>
+        <div class="meetup-description">
+          <div v-html="event.description"></div>
+          <div class="meetup-details">
+            <div class="meetup-date"> <b>When:</b> {{ event.parsed_date }} </div>
+            <div v-if="event.venue" class="meetup-location"> <b>Where:</b> {{ event.venue.name }} </div>
+            <div v-else> <b>Where:</b> TBD </div>
+            <div class="meetup-who"> <b>Who:</b> {{ event.yes_rsvp_count }} already going</div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+//import node module dependencies
+import jsonp from 'jsonp';
+
+//import utility dependencies
+import { parameterize, formatDate } from '../utils/component-utils.js';
+
+
+const DESIRED_HACK_NIGHTS = 4;
+const HACK_NIGHT = "[Everyone] Full Stack Hack Night";
+const EVENT_URL = "http://api.meetup.com/newhavenio/events?";
+const PARAMS = {
+  page: 20,
+  status: 'upcoming',
+}
+               
+export default {
+  data: function() {
+    return {
+      events: [],
+    };
+  },
+  methods: {
+    getEvents: function() {
+      var requestUrl = EVENT_URL + parameterize(PARAMS);
```

Comment:
Why not add these to the `EVENT_URL` since they never change.

If they _do_ change then it might be worth looking into one of the supported HTTP libs which support URL Templating.

```js
const EVENT_URL = "//api.meetup.com/newhavenio/events{?page,status}";
```

Reply:
I could go either way, I wanted to leave the door open to changes down the line is people wanted to request more pages or whatnot, and i thought that it was more easily understood for others to change params from an object rather than params inside a template string.

could go either way, I don't feel strongly about it.